### PR TITLE
configure onboarding navigation

### LINKED ISSUE #101: Configurable onboarding navigation
- [x] it should be configurable if it is shown horizontally or vertically (in the ahoiconfig)

check if it already exists in vega-lite demos (horizon graph or scatterchart)

### PR DESCRIPTION
**Pull request by dvsilviyageo**
Closes #101 

Summary:
* Onboarding navigation can be configured to be either horizontal or vertical
* By default it is vertical.

Screencast:

https://user-images.githubusercontent.com/104566246/203765459-a9faa58f-5452-4757-be57-01fa531bff22.mp4

### CODE REVIEW

**Comment by dvdanielamoitzi** [comment]
Problem: the transition is not working for the horizontal layout @dvdanielamoitzi

File: packages/core/src/components/OnboardingNavigation.svelte [hunk 1]
```diff
@@ -185,14 +217,19 @@
...
     justify-content: center;
     cursor: pointer;
     transition: opacity 0.5s ease, bottom 0.5s ease;
-    width: 80px;
-    /* bottom: 20px; */
-    bottom: 80px;
+    /* width: 80px; */
+
+    /* bottom: 80px; */
+    /* bottom: 30px; */
```
Comment (dvdanielamoitzi) [inline packages/core/src/components/OnboardingNavigation.svelte:230]:
```suggestion
```

File: packages/core/src/components/OnboardingNavigation.svelte [hunk 2]
```diff
@@ -185,14 +217,19 @@
   .visahoi-navigation-marker-container {
     position: absolute;
     display: flex;
-    flex-direction: column;
+    /* flex-direction: column; vertical */
+    /* flex-direction: row; horizontal */
```
Comment (dvdanielamoitzi) [inline packages/core/src/components/OnboardingNavigation.svelte:221]:
```suggestion
```

File: packages/core/src/components/OnboardingNavigation.svelte [hunk 3]
```diff
@@ -221,4 +258,23 @@
...
+    margin-right: 40px;
+  }
+
+  .navigation-right {
+    margin-right: 30px;
+  }
+
+  .horizontal {
+    right: var(--bottom-height);
+    /* transition: opacity 0.5s ease, right 0.5s ease; */
```
Comment (dvdanielamoitzi) [inline packages/core/src/components/OnboardingNavigation.svelte:272]:
```suggestion
```

File: packages/core/src/components/OnboardingNavigation.svelte [hunk 4]
```diff
@@ -221,4 +258,23 @@
...
+    bottom: 0;
+  }
+
+  .vertical {
+    bottom: var(--bottom-height);
+    /* transition: opacity 0.5s ease, bottom 0.5s ease; */
```
Comment (dvdanielamoitzi) [inline packages/core/src/components/OnboardingNavigation.svelte:278]:
```suggestion
```

File: packages/core/src/components/NavigationMarker.svelte [hunk 5]
```diff
@@ -130,7 +130,7 @@
     align-items: center;
     padding: 8px;
   }
-  .visahoi-marker-navigation-item {
+  /* .visahoi-marker-navigation-item {
```
Comment (dvdanielamoitzi) [inline packages/core/src/components/NavigationMarker.svelte:133]: you can remove this codeblock when it's commented out anyways

**Review by dvdanielamoitzi — CHANGES_REQUESTED** [review]

**Comment by dvdanielamoitzi** [comment]
As already discussed, can you please position the "enter edit mode" btn like this?
image

**Comment by dvdanielamoitzi** [comment]
When I have the horizontal navigation, i can somehow click the "right" btn even if I already selected the first point (and the arrow btn is correctly disabled). But when I click it, it jumps to the previous marker (actually nothing at all should happen)

[Screencast from 2022-11-28 10:48:46.webm](https://user-images.githubusercontent.com/57343176/204247137-97f38ecd-b3a1-4036-af60-71e116193e54.webm)

**Comment by dvsilviyageo** [comment]
This branch is outdated all the changes and in this branch is refracted and added to other branch 'sgeo/configurable-onboarding-navigation'

**Comment by dvsilviyageo** [comment]
Fixed in #252

**Pull request closed without merging**